src/pages/[platform]/build-a-backend/add-aws-services/analytics/record-events/index.mdx

```diff
@@ -309,6 +309,7 @@ Events have default configuration to flush out to the network every 30 seconds.
 
 ```json
 {
+  "UserAgent": "aws-amplify-cli/2.0",
```

Comment:
Q: Was there an issue with this missing? I think the user agent will look different if using Gen2. It might be better to just omit it if this wasn't causing an issue.